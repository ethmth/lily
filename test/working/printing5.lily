
print(5 / 2)
print(6 / 2)
print(5.0 / 2.0)

if (4 == 4 and 5 == 5):
    print(True)