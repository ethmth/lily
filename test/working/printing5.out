2
3
2.500000
true